diffkemp/simpll/ModuleComparator.cpp

```diff
@@ -139,9 +162,18 @@ void ModuleComparator::compareFunctions(Function *FirstFun,
                                                        controlFlowOnly,
                                                        showAsmDiffs,
                                                        DI, this);
-            if (fCompSecond.compare() == 0) {
+            result = fCompSecond.compare();
+
+            DEBUG_WITH_TYPE(DEBUG_SIMPLL, decreaseDebugIndentLevel());
```

Comment:
There remains one problem, this loop can be terminated by the `break` statement on line 157. This would cause the indent not to be decreased correctly. I'd suggest putting one additional decrease before the break occurs.

Reply:
Right, missed this one after I moved the decrease into the loop. Should be corrected now.